.github/workflows/kind-e2e.yaml

```diff
@@ -87,9 +87,9 @@ jobs:
       fail-fast: false # Keep running if one leg fails.
       matrix:
         k8s-version:
-        - v1.25.x
         - v1.26.x
         - v1.27.x
+        - v1.28.x
```

Comment:
This currently won't work - you'll need to make a PR here to add it

https://github.com/chainguard-dev/actions/blob/6866fa36bf0235fffb87be13f72996d0d8f1bdf7/setup-kind/action.yaml#L128-L130

Reply:
the node image sha for v1.28 is mentioned in the kind release notes 

see:  https://github.com/kubernetes-sigs/kind/releases/tag/v0.20.0

Reply:
I have created the pull request should I remove it for now and then commit?

Reply:
I think it's ok to wait for the chainguard changes

Reply:
chainguard changes merged

Reply:
Now did not need to remove this?